.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-pnpm lint-staged
\ No newline at end of file
+pnpm lint
```

Comment:
what about the other commands we are using in lint-staged, like prettier? are these now skipped?
Do we still need the lint-staged dependency?